op-proposer/flags/flags.go

```diff
@@ -106,6 +133,7 @@ var optionalFlags = []cli.Flag{
 	WaitNodeSyncFlag,
 }
 
+// init appends RPC, logger, metrics, profile, and tx manager flags to local flags.
```

Comment:
nit: Documenting `init` is not useful as it's called by the go runtime automatically (a la static initialization), rather than a function to be explicitly called by humans.